public/video-ui/src/constants/videoEditValidation.js

```diff
@@ -1,16 +1,18 @@
 const validate = (values) => {
   const errors = {};
-  if (!values.title) {
-    errors.title = 'Required'
-  }
-  if (!values.category) {
-    errors.category = 'Required'
-  }
-  if (!values.posterImage) {
-    errors.posterImage = 'Required'
-  }
-  if (!values.duration) {
-    errors.duration = 'Required'
+  if (values) {
```

Comment:
Would this be nicer just to have a single `if (!values){return false}` at the top?